info.md

```diff
@@ -85,6 +85,7 @@ Currently the following service providers are supported:
 ### New Zealand
 
 - [Auckland](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/aucklandcouncil_govt_nz.md)
+- [Wellington](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/wellington_govt_nz.md)
```

Comment:
Please add the source also in file README.md.

Reply:
done